account_financial_report/report/aged_partner_balance_xlsx.py

```diff
@@ -20,67 +21,84 @@ def _get_report_name(self, report, data=False):
             report_name = report_name + suffix
         return report_name
 
-    def _get_report_columns(self, report):
-        if not report.show_move_line_details:
-            return {
-                0: {"header": _("Partner"), "field": "name", "width": 70},
-                1: {
-                    "header": _("Residual"),
```

Comment:
Residual and current columns should be respected, no matter the configuration.

Reply:
Done